core/chaincode/lifecycle/lifecycle.go

```diff
@@ -316,28 +316,29 @@ type ExternalFunctions struct {
 // CheckCommitReadiness takes a chaincode definition, checks that
 // its sequence number is the next allowable sequence number and checks which
 // organizations have approved the definition.
-func (ef *ExternalFunctions) CheckCommitReadiness(chname, ccname string, cd *ChaincodeDefinition, publicState ReadWritableState, orgStates []OpaqueState) (map[string]bool, error) {
+func (ef *ExternalFunctions) CheckCommitReadiness(chname, ccname string, cd *ChaincodeDefinition, publicState ReadWritableState, orgStates []OpaqueState) (map[string]bool, map[string][]string, error) {
```

Comment:
Since there are now multiple maps returned, I'd suggest to name each of them here in the function definition.

For guidance see:
https://go.dev/doc/effective_go#named-results
https://gist.github.com/adamveld12/c0d9f0d5f0e1fba1e551